test/test_helper.rb

```diff
@@ -13,7 +13,12 @@
 
 ActiveRecord::Migration.verbose = false
 ActiveRecord::Base.logger = Logger.new(nil)
-ActiveRecord::Migrator.migrate(File.expand_path('../dummy/db/migrate', __FILE__))
+if Rails.version == '5.2.0'
```

Comment:
What happens with Rails 5.2.1?  This change probably needs to happen for all versions >= 5.2.0

Reply:
Hello @olbrich you're right, I'll commit the fix.